.dockerignore

```
@@ -0,0 +1,203 @@
+.gitignore
+client.http
+docker-compose.yml
+Dockerfile
+Makefile
+README.md
```

Tem que ajustar o `.gitignore` tbm. O binário acabou sendo commitado.

```suggestion
README.md
build/*
```